packages/xo-server/src/xo-mixins/resource-sets.js

```diff
@@ -371,6 +373,9 @@ export default class {
 
     if (resourceSetId != null) {
       await this.allocateLimitsInResourceSet(resourcesUsage, resourceSetId)
+      $defer.onFailure(() =>
```

Comment:
Should we allocate in case of failure of this [case](https://github.com/vatesfr/xen-orchestra/pull/4861/files#diff-7b3fe1af863319f71198ac50ea887c68L380)?

Reply:
Yes, thanks!

Reply:
What if we start be releasing resources first?

Reply:
Why?

Reply:
Never mind, my suggestion is not pertinent.